features/_your_results/tax_relief_warning.feature

```diff
@@ -8,54 +8,54 @@ Feature: Display Tax Relief Warning
-      | 121.14 | per week         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            | tax-relief-and-your-workplace-pension | tax relief and your workplace pension |
+      | 239.00 | per week         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            | tax-relief-and-your-workplace-pension | tax relief and your workplace pension |
 
     @welsh
     Examples:
       | salary | salary_frequency | warning_message                                                                                                         | tax_relief_link_text                     | tax_relief_link |
       | 11000  | y flwyddyn       | Os nad ydych chi'n talu treth incwm ar eich enillion, cewch ostyngiad treth yn unig ar eich cyfraniadau | ostyngiadau treth ar eich pensiwn gweithle | gostyngiad-treth-ach-pensiwn-gweithle |
 
   @no-javascript
-  Scenario Outline: Earning £11,500 per year
+  Scenario Outline: Earning £12,500 per year
     And my salary is "<salary>" "<salary_frequency>"
     And I progress to the results page
     Then I should NOT see tax relief "<warning_message>"
 
     Examples:
       | salary | salary_frequency | warning_message |
-      | 11850  | per year         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
-      | 987.50 | per month        | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
-      | 911.54 | per 4 weeks      | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
-      | 227.88 | per week         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
+      | 12500  | per year         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
+      | 1042   | per month        | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
+      | 960    | per 4 weeks      | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
+      | 240    | per week         | If you don’t pay income tax on your earnings, you will only receive tax relief on your pension contributions            |
 
     @welsh
     Examples:
       | salary | salary_frequency | warning_message |
       | 11850  | y flwyddyn       | Os nad ydych chi'n talu treth incwm ar eich enillion, fyddwch chi ddim ond yn cael gostyngiad treth ar eich cyfraniadau |
 
   @no-javascript
-  Scenario Outline: Earning above £11,500 per year
```

Comment:
This was wrong right? (originally should be £11,850 if I did understand it right)

Reply:
Yes, it was originally wrong, good spot 👀